views/templates/admin/marketing_step0/buy_step0.tpl

```diff
@@ -10,84 +10,71 @@
 * @copyright 2014-2015 (c) Axalone France
 * @license   http://opensource.org/licenses/GPL-3.0  GNU General Public License, version 3 (GPL-3.0)
 *}
-{if is_array($smarty_fax_tickets) || is_array($smarty_sms_tickets)}
+
 <form id="buy_form" class="defaultForm form-horizontal adminmarketingx" action="index.php?controller={if (!empty($credential_fax) || !empty($credential_sms))}AdminMarketingBuy&token={Tools::getAdminTokenLite('AdminMarketingBuy')|escape:'html':'UTF-8'}{else}AdminMarketingInscription&token={Tools::getAdminTokenLite('AdminMarketingInscription')|escape:'html':'UTF-8'}{/if}" method="post" enctype="multipart/form-data" novalidate>
-	<div class="panel" id="fieldset_0">
+	<div class="panel" id="fieldset_0" style="min-width: 630px; width: 50%; padding-bottom: 3px">
```

Comment:
Instead of using inline style, you should move your style definition into a CSS file.